cards/src/main/resources/cards/custom/group0/other_death_knight/koft/spell_permafrost.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "Permafrost",
+  "baseManaCost": 4,
+  "type": "SPELL",
+  "heroClass": "SPIRIT",
+  "rarity": "EPIC",
+  "description": "Freeze a minion. Permanently,",
```

Comment:
Also might be more appropriate to implement as an attribute aura or a trigger that always freezes at the start of the turn.